tests/functional/features/patient_access_update.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Patient Access (Update)
+	Update a PDS record as a patient
+
+	Scenario: Patient cannot update another patient
+		Given I am a P9 user
+		And scope added to product
+
+		When I update another patient's PDS record
+
+		Then I get a 403 HTTP response code
+		And Patient cannot perform this action is at issue[0].details.coding[0].code in the response body
+ 
+	Scenario: Patient update uses incorrect path
+		Given I am a P9 user
+		And scope added to product
+
+		When I update another patient's PDS record using an incorrect path
+
+		Then I get a 403 HTTP response code
+		And Patient cannot perform this action is at issue[0].details.coding[0].code in the response body
```

Comment:
This fails with the new `assert` suggestion in conftest.py. The path needs changing to 
```suggestion
		And Patient cannot perform this action is at issue[0].details.coding[0].display in the response body
```